fvm/meter/meter.go

```diff
@@ -1,17 +1,57 @@
 package meter
 
+type MetringOperationType uint
+
+// continuation of cadence.ComputationKind
+const (
+	// [2_000, 3_000) reserved for the FVM
+	_ = iota + 2_000
```

Comment:
Should this be typed as `ComputationKind`? e.g:
```suggestion
	_ cadence.ComputationKind = iota + 2_000
```

And then use the key-type of the intensities maps as also `cadence.ComputationKind`? e.g:
```go
type Meter struct {
    ...
    computationIntensities map[cadence.ComputationKind]uint
    ...
}
```

Reply:
Nifty! Yes, this is definitely nicer.